frontend/src/app/login/login.component.html

```diff
@@ -27,17 +27,8 @@ <h1>Login</h1>
         <input id="password" #password name="password"
                [formControl]="passwordControl"
                (focus)="this.error=null"
-               matInput placeholder="" [type]="hide ? 'password' : 'text'"
+               matInput placeholder="" [type]="password"
```

Comment:
I don't really like the removal of "show password" functionality, but i'm not sure what else could be done, so let's leave this change in